### PR TITLE
feat(dotnet6): SAM template for .NET 6 C# Quick Start: Scheduled Events

### PR DESCRIPTION
**Pull request by mzambani**
*Issue #, if available:* #200

*Description of changes:*
DotNet6 template for SAM C# Quick Start: Scheduled Events

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ulili5 — APPROVED** [review]
Great job @mzambani this looks good to me

**Review by ulili5 — APPROVED** [review]

**Comment by mingkun2020** [comment]
Thank you for your contribution! We really appreciate it. We will take time to review the pr.

**Review by jfuss — APPROVED** [review]

**Comment by jfuss** [comment]
Looks reasonable to me. @mzambani Can you handle the conflicts?

File: manifest-v2.json [hunk 1]
```diff
@@ -91,6 +91,13 @@
...
     },
     {
+      "directory": "dotnet6/cookiecutter-aws-sam-quick-start-cloudwatch-events-dotnet",
+      "displayName": "Quick Start: Scheduled Events",
+      "dependencyManager": "cli-package",
+      "appTemplate": "quick-start-cloudwatch-events",
+      "packageType": "Zip",
+      "useCaseName": "Scheduled task"
+    },
       "directory": "dotnet6/cookiecutter-aws-sam-quick-start-from-scratch-dotnet",
```
Comment (sostenesbarros) [inline manifest-v2.json:101]: Missing open curly braces.

**Pull request merged**